src/main/java/de/themorpheus/edu/cli/Command.java

```diff
@@ -0,0 +1,24 @@
+package de.themorpheus.edu.cli;
+
+import com.beust.jcommander.JCommander;
+
+import lombok.Data;
+
+@Data
+public class Command<T> {
+	
+	private String name;
+	private String description;
+	private String[] alias;
+	private T args;
+	private CommandExecuter<JCommander, Options, T> executer;
+	
+	public Command(String name, String description, CommandExecuter<JCommander, Options,T> executer,  Class<T> args, String... alias) throws InstantiationException, IllegalAccessException {
```

Comment:
Initialize "args" in the variable and use final variables with @RequiredArgsConstructor